### PR TITLE
fix caseid selection

### PR DESCRIPTION
**Pull request by benrudolph**
@czue another minor fix from running some dry runs. i'm having a hard time figuring out how to proceed on this migration. here are a couple thoughts:
- Ask VL to do QA of export conversions. This actually becomes fairly difficult because there is no way to create a domain that uses old exports. Probably doable on an old domain, but it would only test the lazy export conversion not the bulk migration (uses a lot of the same code though)
- Pick and handful of random domains, do the migration and incrementally address any issues that should arise
- Just do the whole thing (seems scary)

The most common failure mode of this is a selection of a system property does not persist to the migrated export or that we failed to include a system property that a client really wanted

### CODE REVIEW

**Comment by benrudolph** [comment]
cc: @esoergel @orangejenny

File: corehq/apps/export/conversion_mappings.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
 CASE_PROPERTY_MAPPING = {
     ("id", None): ([PathNode(name="number")], None),
-    ("_id", None): ([PathNode(name='caseid')], None),
+    ("_id", None): ([PathNode(name='_id')], None),
```
Comment (czue) [inline corehq/apps/export/conversion_mappings.py]: was `caseid` just wrong?

**Comment by czue** [comment]
Of those options 

- Pick and handful of random domains, do the migration and incrementally address any issues that should arise

seems like the one with the least risk and friction to me (when you account for tradeoffs between both). how are you finding these issues in your dry runs currently?

**Comment by czue** [comment]
i just wanted to understand this change before merging, but feel free to self merge once you answer the question.

File: corehq/apps/export/conversion_mappings.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/export/conversion_mappings.py]: yep, caseid is the label, `_id` is where the actual id lives on the doc

Comment (benrudolph) [inline corehq/apps/export/conversion_mappings.py]: see this: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/export/system_properties.py#L252

**Comment by benrudolph** [comment]
dry run creates the schema and then logs all the skipped columns and tables to a file. then i look over the file and see if all the ones skipped makes sense. usually there are very few that are skipped. but if the user just hit select all then things `_rev` will be skipped

**Comment by benrudolph** [comment]
ok i agree with that, i'll try and put together a plan based on that strategy

**Pull request merged**